<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettingsDialogClass</class>
 <widget class="QDialog" name="GeneralSettingsDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeneralSettingsDialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>480</width>
     <height>380</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Startup</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>450</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Program Startup</string>
     </property>
     <widget class="QCheckBox" name="cShowStationIdAtStartup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>305</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Show station ID on startup</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cShowTimeAtStartup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>370</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Show the PC time check to confirm time is correct</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>450</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Custom Folder Labels</string>
     </property>
     <widget class="QLineEdit" name="cFolder_1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>25</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Folder 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Folder 2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cFolder_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Folder 3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cFolder_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>75</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>105</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Folder 4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cFolder_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Folder 5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cFolder_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>125</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Addressing</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>450</width>
       <height>330</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Profiles</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>450</width>
       <height>330</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Printing</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>450</width>
       <height>330</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Miscellaneous</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>450</width>
       <height>330</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cOK">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>395</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cCancel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>395</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cOK</sender>
   <signal>clicked()</signal>
   <receiver>GeneralSettingsDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cCancel</sender>
   <signal>clicked()</signal>
   <receiver>GeneralSettingsDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
